pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "nixops_vbox"
+version = "1.0"
+description = "NixOps plugin for VirtualBox"
+authors = ["Amine Chikhaoui <amine.chikhaoui91@gmail.com>"]
+include = [ "nixops_vbox/nix/*.nix" ]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+nixops = {git = "https://github.com/adisbladis/nixops.git", rev = "poetry2nix-plugins"}
```

Comment:
Once https://github.com/NixOS/nixops/pull/1256/ merges this should point to master.